frontend/exporter/Cargo.toml

```diff
@@ -23,3 +23,4 @@ tracing.workspace = true
 paste = "1.0.11"
 extension-traits = "1.0.1"
 lazy_static = "1.4.0"
+deterministic-hash = "1.0.1"
```

Comment:
What hash function is this? Can't we use something better? This looks like a pretty odd dependency that hasn't been updated in over 2 years.

Reply:
It is just a wrapper around other hashers: it's simply "normalizing" some types (i.e. `usize` is mapped to `u64` regardless of the platform) before calling the inner hasher, so that the hash is deterministic over various plateforms.

Reply:
Looking at what this is doing we should rather copy the few lines that are needed here. I want to avoid getting into a dependency hell here and this doesn't seem worth it.

Reply:
Makes sense, that's done in e82e74a 🫡